charts/cni-metrics-helper/Chart.yaml

```diff
@@ -1,23 +1,19 @@
-apiVersion: v2
+apiVersion: v1
```

Comment:
Do we need to change this to upload to eks-charts or just because we don't mandate the helm version to be 3 and above for installation?

Reply:
I was thinking about helm2 compatibility, but since this chart is new and being released after we specified helm3 is required, I will put this back to `v2`.